pos_session_report/README.rst

```diff
@@ -0,0 +1,84 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Pos Session Report Detailed
+==============
+
+This module extends the functionality of Pos Session Sumary
+and to allow you to show detailed informations about pos session
+
+Installation
+============
+
+To install this module, you need to:
+
```

Comment:
If there is not special considerations to take in care just write it "normal instalation"